extension/src/shared/view/selection/ColorSchemeSectionForComponentField.tsx

```diff
@@ -94,8 +97,9 @@ export const ColorSchemeSectionForComponentField: FC<ColorSchemeSectionForCompon
                 }
               })
               .filter(isNotNullish) ?? [];
+          const defaultItems: [null, string][] = useNone ? [[null, "なし"]] : [];
           return [
-            [null, "なし"],
+            ...defaultItems,
             ...rules.map((rule): [string, string] => [rule.id, rule.propertyName ?? ""]),
```

Comment:
I guess we also need `rule.legendName ?? rule.propertyName ?? ""` here?

Reply:
Fixed! https://github.com/eukarya-inc/PLATEAU-VIEW-3.0/pull/65/commits/a64d8ce512bc1ef94dafb4f72c9ff1286d12bcc8